copybook4java-codegen-maven-plugin/src/test/java/com/nordea/oss/copybook/codegen/JavaSyntaxChecker.java

```diff
@@ -39,7 +39,12 @@
         StandardJavaFileManager fileManager = javac.getStandardFileManager(null, null, StandardCharsets.UTF_8);
         DiagnosticCollector<JavaFileObject> diagnostics = new DiagnosticCollector<>();
         JavaCompiler.CompilationTask task = javac.getTask(null, new InMemmoryFileManager(javac.getStandardFileManager(null, null, null)), diagnostics, null, null, stringSourceCodes);
-        task.call();
+        try {
+            task.call();
```

Comment:
I guess you still want to re throw the exception if the task fails

Reply:
Yes, please introduce unchecked exception to be thrown here instead of system.out printing.